dev-support/ci/selective_ci_checks.sh

```diff
@@ -521,12 +521,14 @@ function set_outputs() {
     initialization::ga_output basic-checks \
         "$(initialization::parameters_to_json ${BASIC_CHECKS})"
 
-    if [[ "${compose_tests_needed}" == "true" ]] || [[ "${kubernetes_tests_needed}" == "true" ]]; then
+    : ${compile_needed:=false}
```

Comment:
I am not sure I follow this,

Reply:
`: ${compile_needed:=false}`

This assigns default value `false` to the variable `compile_needed`.  If the variable is already set, this is a no-op.